akka-actor/src/main/scala-2.13+/akka/util/ByteString.scala

```diff
@@ -253,6 +252,14 @@ object ByteString {
       buffer.putByteArrayUnsafe(bytes)
     }
 
+    override def copyToArray[B >: Byte](dest: Array[B], start: Int, len: Int): Int = {
+      val copied = math.max(math.min(math.min(len, bytes.length), dest.length - start), 0)
```

Comment:
it's save a `math.max` if we just return a constant `0` in the 'else' - not sure it matters but since we're shaving anyway :smile: